src/QMCHamiltonians/CoulombPBCAA.cpp

```diff
@@ -538,9 +537,10 @@ std::vector<CoulombPBCAA::Return_t> CoulombPBCAA::mw_evalSR_offload(const RefVec
   const auto delta_inv   = caa_leader.rVs_offload->get_delta_inv();
   const auto Zat         = caa_leader.Zat_offload->data();
 
-  auto value_ptr = values_offload.data();
-
   {
+    values_offload.resize(nw);
+    std::fill_n(values_offload.data(), nw, 0);
```

Comment:
Is this initialization strictly required? Can we avoid it by writing the penultimate line of the offload loop below not as a reduction but an assignment?

Reply:
This is strictly required. The zeroing serves the reduction over chunks while the offload only serves one chunk. So it must be done outside the chunk loop.